.github/workflows/addon-checker.yml

```
@@ -1,18 +1,23 @@
     runs-on: ubuntu-latest
-    name: Kodi addon checker
     steps:
-    - name: Checkout
-      uses: actions/checkout@v1
-    - name: Kodi addon checker validation
-      id: kodi-addon-checker
-      uses: xbmc/action-kodi-addon-checker@v1.1
+    - uses: actions/checkout@v4
+    
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)